app/assets/v2/images/tribes/logo-townsquare.svg

```diff
@@ -0,0 +1,424 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
can we optimize this with https://jakearchibald.github.io/svgomg/ ? As a recommendation don't use inkscape they create svg attr and tags out of standards only works with  inkscape, as a free solution I recommend using figma (their export is very easy also) 
But in general I always use https://jakearchibald.github.io/svgomg/ at the end.